package.json

```diff
@@ -15,7 +15,8 @@
     "format": "prettier --write \"**/*.{js,json,md,yml}\"",
     "ci": "npm run build && npm run test-cover && npm run lint",
     "prepublishOnly": "npm run lint && npm test && npm run build",
-    "release": "release-it"
+    "release": "release-it",
+    "prepare": "npm run build"
```

Comment:
I don't think we need this script for our lifecycle?

Reply:
sorry. was experimenting in my fork :)